packages/app/src/components-styled/sewer-chart/sewer-chart.tsx

```diff
@@ -192,6 +194,25 @@ export function SewerChart(props: SewerChartProps) {
       </Box>
 
       <Box position="relative" ref={sizeRef} css={css({ userSelect: 'none' })}>
+        <svg
+          width={width}
+          height={52}
+          role="img"
+          style={{ pointerEvents: 'none' }}
+        >
+          {outlierValues && (
+            <Group left={dimensions.padding.left}>
+              <ScatterPlot
+                data={outlierValues}
+                getX={scales.getX}
+                getY={(_x: SewerChartValue) => 26}
```

Comment:
```suggestion
                getY={() => 26}
```

I suspect that will work too